home/config.xml

```diff
@@ -3,7 +3,7 @@
   <disabledAdministrativeMonitors>
     <string>jenkins.security.s2m.MasterKillSwitchWarning</string>
   </disabledAdministrativeMonitors>
-  <version>2.150.3</version>
+  <version>2.319.2</version>
```

Comment:
Interestingly the Docker image tracks the `lts` tag but this is an explicit version. Should the two be aligned or do we foresee it not being an issue?

Reply:
I noticed that too. It is a bit unclear.

Reply:
Assuming the `jenkins:2.319.2` image exists, maybe this should be used as the base image in `jenkins/Dockerfile` to minimize future conflicts

Reply:
I will check what's available to reduce problem